_posts/2017-01-22-Swift-at-Artsy.md

```diff
@@ -0,0 +1,298 @@
+---
+layout: post
+title: "Postmortem: Swift at Artsy"
+date: 2017-01-22 12:18
+author: orta
+categories: [swift, eigen, eidolon, javascript, emission, reactnative]
+series: React Native at Artsy
+---
+
+<center>
+<img src="/images/swift-in-rn/swift-in-react-native.svg" style="width:300px;">
+</center>
+
+Swift became public in June 2014, by August we had started using it in Artsy. By October, we had [Swift in production][eidolon-postmortem] channeling hundreds of thousands of dollars in auction bids. 
+
+It is pretty obvious that Swift is the future of native development on Apple platforms. It was a no-brainer to then build an [appleTV app][emergence] in Swift, integrated Swift-support into our key app Eigen and built non-trivial parts of that [application in Swift][live-a].
+
+We first started experimenting with with React Native in February 2016, and by August 2016, we announced that [Artsy moved to React Native][artsy-rn] effectively meaning new code would be in JavaScript from here onwards.
+
+We're regularly asked _why_ we moved, and it was touched on briefly in our announcement but I'd like to dig in to this and try to cover a lot of our decision process. So, if you're into understanding why a small team of iOS developers with decades of experience switched to JavaScript, read on. 
+
+This post will cover: What are Artsy's apps?, Swifts Positives and Negatives for us, React Native, and Our 1-year Summary.
+
+<!-- more -->
+
+We were finding that our current patterns of building apps were not scaling as the team and app scope grew. Building anything new inside Eigen barely re-used existing native code, and it was progressively taking longer and longer to build features. App and test build times were increasing, it would take 2 iOS engineers to build a feature in a similar time-frame as a single web engineer. Our iOS engineers have a lot of experience across many platforms, are well versed in best practices and understand the value of building better tools to make it faster 
```

Comment:
I don't get the intended point of the last sentence in this paragraph. Is it, "our iOS engineers are super smart, so it's silly it took them so long"? "Because we understand the value of building tools, we thought it was important to fix this"? Something else?